traffic_portal/app/src/app.js

```diff
@@ -474,7 +475,7 @@ trafficPortal.factory('authInterceptor', function ($rootScope, $q, $window, $loc
             if (rejection.status === 401) {
                 $rootScope.$broadcast('trafficPortal::exit');
                 userModel.resetUser();
-                if (url == '/login' || $location.search().redirect) {
+                if (url == '/login' || url =='/sso' || $location.search().redirect) {
```

Comment:
nitpick: I know we aren't doing this already (as evidenced by `url == '/login'`), but, in general, it's better to compare JS values with "triple-equals" (`===`/`!==`) .

Reply:
done